src/main/java/com/ixortalk/autoconfigure/oauth2/auth0/mgmt/api/Auth0Roles.java

```diff
@@ -72,22 +74,27 @@ public void assignRolesToUser(String email, Set<String> roleNamesToAssign) {
             return;
         }
 
-        this.auth0ManagementAPI
-                .assignRolesToUser(
-                        this.auth0ManagementAPI.listUsersByEmail().get(email).getId(),
-                        roleNamesToAssign.stream().map(roleName -> this.auth0ManagementAPI.listRolesByName().get(roleName).getId()).collect(toList())
-                );
+        auth0ManagementAPI.getUserByEmail(email)
```

Comment:
Below two calls won't fail anymore when a user with the given email cannot be found.  I don't think it's desirable for these calls not to fail when the user cannot be found.